README.md

```diff
@@ -35,6 +36,15 @@ In this setup where both standalone and server mode are available, the login is
 only required to manage documents on the server or validate against the
 [validator service](https://github.com/secvisogram/csaf-validator-service).
 
+To build the application run:
+```shell
+./bradlew clean build
```

Comment:
Is that really `./bradlew`? Or is that `./gradlew`? Sorry - I didn't had time to test that at my dev environment...